{"hierarchy":{"paths":[["doc:\/\/com.auth0.Auth0\/documentation\/Auth0","doc:\/\/com.auth0.Auth0\/documentation\/Auth0\/AuthenticationError"]]},"kind":"article","topicSections":[{"anchor":"Instance-Properties","identifiers":["doc:\/\/com.auth0.Auth0\/documentation\/Auth0\/AuthenticationError\/failureReason","doc:\/\/com.auth0.Auth0\/documentation\/Auth0\/AuthenticationError\/helpAnchor","doc:\/\/com.auth0.Auth0\/documentation\/Auth0\/AuthenticationError\/recoverySuggestion"],"title":"Instance Properties","generated":true}],"sections":[],"metadata":{"modules":[{"name":"Auth0"}],"title":"LocalizedError Implementations","roleHeading":"API Collection","role":"collectionGroup"},"identifier":{"interfaceLanguage":"swift","url":"doc:\/\/com.auth0.Auth0\/documentation\/Auth0\/AuthenticationError\/LocalizedError-Implementations"},"variants":[{"paths":["\/documentation\/auth0\/authenticationerror\/localizederror-implementations"],"traits":[{"interfaceLanguage":"swift"}]}],"schemaVersion":{"minor":3,"major":0,"patch":0},"references":{"doc://com.auth0.Auth0/documentation/Auth0/AuthenticationError/recoverySuggestion":{"kind":"symbol","identifier":"doc:\/\/com.auth0.Auth0\/documentation\/Auth0\/AuthenticationError\/recoverySuggestion","abstract":[],"type":"topic","url":"\/documentation\/auth0\/authenticationerror\/recoverysuggestion","role":"symbol","fragments":[{"text":"var","kind":"keyword"},{"text":" ","kind":"text"},{"text":"recoverySuggestion","kind":"identifier"},{"text":": ","kind":"text"},{"text":"String","preciseIdentifier":"s:SS","kind":"typeIdentifier"},{"text":"?","kind":"text"}],"title":"recoverySuggestion"},"doc://com.auth0.Auth0/documentation/Auth0/AuthenticationError/helpAnchor":{"role":"symbol","fragments":[{"text":"var","kind":"keyword"},{"text":" ","kind":"text"},{"text":"helpAnchor","kind":"identifier"},{"text":": ","kind":"text"},{"text":"String","preciseIdentifier":"s:SS","kind":"typeIdentifier"},{"text":"?","kind":"text"}],"url":"\/documentation\/auth0\/authenticationerror\/helpanchor","type":"topic","identifier":"doc:\/\/com.auth0.Auth0\/documentation\/Auth0\/AuthenticationError\/helpAnchor","kind":"symbol","title":"helpAnchor","abstract":[]},"doc://com.auth0.Auth0/documentation/Auth0/AuthenticationError/failureReason":{"url":"\/documentation\/auth0\/authenticationerror\/failurereason","title":"failureReason","role":"symbol","type":"topic","abstract":[],"kind":"symbol","fragments":[{"kind":"keyword","text":"var"},{"kind":"text","text":" "},{"kind":"identifier","text":"failureReason"},{"kind":"text","text":": "},{"preciseIdentifier":"s:SS","kind":"typeIdentifier","text":"String"},{"kind":"text","text":"?"}],"identifier":"doc:\/\/com.auth0.Auth0\/documentation\/Auth0\/AuthenticationError\/failureReason"},"doc://com.auth0.Auth0/documentation/Auth0/AuthenticationError":{"abstract":[{"type":"text","text":"Represents an error during a request to the Auth0 Authentication API."}],"fragments":[{"text":"struct","kind":"keyword"},{"text":" ","kind":"text"},{"text":"AuthenticationError","kind":"identifier"}],"identifier":"doc:\/\/com.auth0.Auth0\/documentation\/Auth0\/AuthenticationError","type":"topic","title":"AuthenticationError","navigatorTitle":[{"text":"AuthenticationError","kind":"identifier"}],"kind":"symbol","url":"\/documentation\/auth0\/authenticationerror","role":"symbol"},"doc://com.auth0.Auth0/documentation/Auth0":{"role":"collection","abstract":[{"text":"SDK for Apple platforms.","type":"text"}],"type":"topic","identifier":"doc:\/\/com.auth0.Auth0\/documentation\/Auth0","kind":"symbol","url":"\/documentation\/auth0","title":"Auth0"}}}